app/routes.py

```diff
@@ -1 +1,254 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint,jsonify, abort,make_response,request
+from app import db
+import requests
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+
+task_list_bp = Blueprint("tasks", __name__, url_prefix ="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix = "/goals")
+
+@task_list_bp.route("", methods = ["POST"])
+def create_tasks():
+    request_body = request.get_json()
+    print(request_body)
+    print("************")
+    
+    if (not "title" in request_body) or (not "description" in request_body):
+        return{
+            "details":"Invalid data"
+        }, 400
+    try:
+        new_task = Task.from_dict(request_body)
+        db.session.add(new_task)
+        db.session.commit()
+
+        #message = f"Task {new_task.title} successfully created"
+        return make_response(jsonify({"task":new_task.task_dict()}), 201)
+    
+    except KeyError as e:
+        abort(make_response("Invalid request. Missing required value: {e}"), 400)
+
+@task_list_bp.route("/<id>", methods = ["GET"])
+def get_one_saved_task(id):
+    task = validate_task(id)
+    return jsonify({"task":task.task_dict()}), 200
+ 
+@task_list_bp.route("", methods = ["GET"])
+def get_all_saved_tasks():
+    sort_query=request.args.get("sort")
+    tasks_query=Task.query
+    
+    if sort_query =="asc":
+        tasks_query = Task.query.order_by(Task.title.asc())
+    elif sort_query =="desc":
+        tasks_query = Task.query.order_by(Task.title.desc())
+    
+    tasks = tasks_query.all()
+
+    tasks_response =[task.task_dict() for task in tasks]
+
+    return (jsonify(tasks_response)),200
+        
+    
+    
+def validate_task(task_id):
```

Comment:
Not too big of an issue since you are only ever using validate_task to validate your tasks, but it's never a bad idea to make it a generic function in case you have other models that need to be validated later! Also, since it is a helper function, best practice would either have it placed before your routes or in its own helper file!